File: AkitaModel/BgNhis/NhifPack.cs
using System.Collections.Generic;

namespace Skyware.Lis.AkitaModel.BgNhis;

/// <summary>
/// NHIF (НЗОК) пакет дейности
/// </summary>
public class NhifPack
{

    /// <summary>
    /// NHIF (НЗОК) code.
    /// </summary>
    public string Code { get; set; }

    /// <summary>
    /// Name, according to NHIF (НЗОК).
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    /// Examinations (Изследвания)
    /// </summary>
    public IList<Examination> Examinations { get; set; }

}

File: AkitaMotelTS/src/BgNhis/NhifPack.ts
/**
 * Skyware.Lis.AkitaModel.BgNhis
 */

import { Examination } from './Examination';

/**
 * NHIF (НЗОК) пакет дейности
 */
export class NhifPack {
  /**
   * NHIF (НЗОК) code.
   */
  code?: string;

  /**
   * Name, according to NHIF (НЗОК).
   */
  name?: string;

  /**
   * Examinations (Изследвания)
   */
  examinations?: Examination[];
}
